examples/client-javascript-test-app/src/main/javascript/support/qunit.js

```diff
@@ -2056,7 +2056,7 @@ function inArray( elem, array ) {
 }
 
 /*
- * Javascript Diff Algorithm
+ * JavaScript Diff Algorithm
```

Comment:
Same here. Just for license sanity we might not want to touch dependent files (but well, probably a bit nit picky)